certificate-policy.md

```diff
@@ -752,6 +752,18 @@ All the physical control requirements specified below apply equally to the Root
 #### 5.1.1 Site location and construction
 The location and construction of the facility housing the CA equipment, as well as sites housing remote workstations used to administer the CAs, shall be consistent with facilities used to house high-value, sensitive information.  The site location and construction, when combined with other physical security protection mechanisms such as guards, high security locks, and intrusion sensors, shall provide robust protection against unauthorized access to the CA equipment and records.
 
+When a cryptographic module housing a CA private key is physically moved to a new site location, a CA operating under this policy will comply with the following procedures.
```

Comment:
See my comments in the issue.  When would this occur?

Reply:
This could occur when physically moving operations to a new data center. A new data center would need an ATO before a key is physically transferred making this potentially moot.